src/pg_statviz/libs/plot.py

```diff
@@ -13,15 +13,15 @@
 
 def setup():
     for f in ["NotoSans-Regular.ttf", "NotoSans-SemiBold.ttf"]:
-        f = importlib.resources.path("pg_statviz.libs", f)
+        f = importlib.resources.files("pg_statviz.libs").joinpath(f)
         fnt.fontManager.addfont(f)
     plt.rcParams['font.family'] = 'Noto Sans'
     plt.rcParams['font.size'] = 12
-    im = plt.imread(importlib.resources.path("pg_statviz.libs",
-                                             "pg_statviz.png"))
+    base_image_path=importlib.resources.files("pg_statviz.libs").joinpath("pg_statviz.png")
+    im = plt.imread(str(base_image_path))
     height = im.shape[0]
     fig = plt.figure(figsize=(19.2, 10.8))
-    fig.figimage(im, 5, (fig.bbox.ymax - height - 6), zorder=3)
+    fig.figimage(im, 5, (fig.get_window_extent().ymin - height - 6), zorder=3)
```

Comment:
Why change this? It broke the top left logo... Also it has nothing to do with importlib.